### DISCUSSION TITLE
Unable to find BlackHole after Build Xcode 13.1

### DISCUSSION THREAD

**Discussion by jophigit** [ExistentialAudio/BlackHole#491] [Q&A] [answered]
Refering to: https://github.com/ExistentialAudio/BlackHole/wiki/Build-with-Xcode (and others)
I followed the instructions, but I cannot find the folder, where it's being built. Before some time, I already tried it, and I found the build inside of a folder called Build or Product (I'm not sure.) - I have no error messages inside the log.

**Comment by jophigit** [comment]
Ok, I think I found it. Now it's somewhere around here: /Users/[folder]/Library/Developer/Xcode/DerivedData/BlackHole-[anyhash]/Build/Products/

**Comment by devinroth** [comment, marked as answer]
In the latest update of Xcode there’s a menu item under Product called “Show Build Folder In Finder”. Just click on that.

Devin

**Reply by jophigit** [reply]
Thx, Devin! 🤗
